manifests/rules/out/hkp.pp

```diff
@@ -0,0 +1,6 @@
+# @summary allow outgoing hkp connections to gpg keyservers
+class nftables::rules::out::hkp {
+  nftables::rule { 'default_out-hkp':
+    content => 'tcp dport 11371 accept',
```

Comment:
I don't know this module very well, but how does this limit it to outgoing traffic? doesn't this also open it incoming?

Reply:
It parses the chain from t.he resource title. In this case the rule is added to the output chain